core/src/main/scala/org/apache/spark/sql/SnappySession.scala

```diff
@@ -1855,7 +1855,11 @@ object SnappySession extends Logging {
         evaluatePlan(df, session, key.sqlText, key)
       }
     }
```

Comment:
lazy val has performance penalty and I was trying to avoid that especially for the concurrency scenarios.